packages/rx_bloc_cli/lib/src/processors/ios/info_plist_processor.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:rx_bloc_cli/src/extensions/string_buffer_extensions.dart';
+
+import '../common/abstract_processors.dart';
+
+/// String processor used for processing an Info.plist file in the ios
+/// directory found at: ios/Runner/Info.plist
+class InfoPlistProcessor extends StringProcessor {
+  /// Podfile file processor constructor
+  InfoPlistProcessor(super.args);
+
+  @override
+  String execute() {
+    if (input == null) return '';
+    final buffer = StringBuffer(input!);
+
+    //_updateGlobalIOSPlatform(buffer);
```

Comment:
Make sure to clean up any unnecessary comments.

Reply:
Removed